l5kit/l5kit/evaluation/csv_utils.py

```diff
@@ -0,0 +1,246 @@
+import csv
+from itertools import chain
+from typing import Iterator, List, Optional
+
+import numpy as np
+
+MAX_MODES = 3
+
+"""
+These utilities write and read csv with ground-truth and prediction data.
+Both share the first two field (timestamp and track_id) which are used to identify a single record.
+
+GT:
+Single mode future prediction with availabilities (either 1->available or 0->unavailable).
+Header fields have these meanings:
+timestamp, track_id, avail_time_0, avail_time_1, ..., coord_x_time_0, coord_y_time_0, ...
```

Comment:
How about abbreviating this a bit by mentioning there are N timesteps and then writing availability_0, x_0, ... availalbility_N, x_N, y_N